src/components/main/flight/CheckoutWithFlightScreen.js

```diff
@@ -142,254 +93,186 @@ export default class CheckoutWithFlightScreen extends React.Component {
     } = this.state;
 
     const years = this.calcYears();
-    const color = this.colorVariant();
 
     return (
-      <View style={styles.container}>
-        <View style={styles.containerTop}>
-          <View style={styles.buttonBarNav}>
-            {/*Navigation Buttons*/}
-            <Ionicons
-              style={styles.navigationIcon}
-              name="md-arrow-back"
-              onPress={() => this.props.navigation.goBack()}
-            />
-            <Text style={styles.midBlueText}>
-              {i18n.t("FLIGHT")} {flightChars} {flightNums}
-            </Text>
-            <FontAwesome
-              style={styles.navigationIcon}
-              name="user-circle-o"
-              onPress={() => this.handleUserRedirect()}
-            />
-          </View>
-          <View style={styles.topText}>
-            {/*CO2 footprint*/}
-            <Text style={styles.bigGreyText}>{footprint}</Text>
-            <View style={styles.alignSubScript}>
-              <Text style={styles.midGreyText}>{i18n.t("METRIC TONS")} CO</Text>
-              <Text
-                style={{
-                  fontSize: 10,
-                  lineHeight: 30,
-                  color: COLORS.darkgrey,
-                }}>
-                2
+      <View style={styles.backDrop}>
+        <View style={styles.innerView}>
+          <View style={styles.top}>
+            {/* Flight Number and Carbon Emissions */}
+            <View>
+              <Text style={styles.paragraph}>
+                {i18n.t("Flight Number")} {flightChars} {flightNums}
               </Text>
+              <Text style={styles.emissions}>{footprint}</Text>
+              <Text style={styles.emissionsUnits}>{i18n.t("tons of CO2")}</Text>
             </View>
-          </View>
-          <View style={styles.iterateGroup}>
-            {/*Subtract tree*/}
-            <TouchableOpacity
-              style={[
-                styles.iterators,
-                {
-                  backgroundColor:
-                    this.state.treeNum == 0 ? COLORS.grey : COLORS.lightgreen,
-                },
-              ]}
-              onPress={() => this.handleRemove()}>
-              <Feather
-                style={[
-                  styles.iteratorIcon,
-                  {
-                    color:
-                      this.state.treeNum == 0 ? COLORS.white : COLORS.lightblue,
-                  },
-                ]}
-                name="minus"
-              />
-            </TouchableOpacity>
-            {/*Tree counter*/}
-            <View style={[styles.treeCounter, { backgroundColor: color }]}>
-              <Text style={styles.treeCountText}>{treeNum}</Text>
+
+            {/* Number of Trees */}
+            <View style={styles.treeNumRow}>
+              <View style={styles.treeNumCol}>
+                <Image source={tree} style={styles.treeImg} />
```

Comment:
is it possible to keep the tree image static? Like it can move when changing from 1 digit to 2 but currently for every single number it's jumping around like someone watered it with cocaine

Reply:
The tree is static, its the numbers. I know it's not smooth maybe I can add a transition to make  double digits cleaner

Reply:
Could you try overlaying the numbers and have the tree in an absolute position? So they wouldn't be in the same treeNumRow view. but rather one rendered above another

Reply:
LOL, clean drawings there. I'll give it a try and get back to you.

Reply:
I find that the tree behind the text is not a good design choice. It is distracting and even with adjusted opacity it does not look that good..I think we should repurpose it into the checkout button as an icon or just be done with it all together.  I am pushing what I have for today.